discover/portal/articles/190-managing-users/01-users-and-organizations/02-advanced-user-management.markdown

```diff
@@ -147,14 +147,14 @@ names conflicted with site IDs. In @product-ver@, this property defaults to
 
     users.screen.name.allow.numeric=true
 
-This means you're free to set a user screen name to *97854*, or whatever other
+This means you're free to set a user screen name to *24601*, or whatever other
```

Comment:
@nealrobbins Out of curiosity, why did you change the number?

Reply:
No, sorry, it's not important. I was trying to get find-and-replace commands to stick.

Reply:
Haha, nice. My working theories were

1. 97854 exactly matches your salary and you didn't want it in the docs.
2. 97854 was your ID number at the penitentiary and you didn't want to see it in the docs.
3. 24601 was your ID number at the penitentiary and you wanted it in the docs.